src/modules/PowerShell/FinOpsHub/FinOpsHub.psm1

```diff
@@ -0,0 +1,193 @@
+data LocalizedData
+{
+    # culture="en-US"
+    ConvertFrom-StringData -StringData @'
+    DeployResourceGroupScope = Deploying hub from template '{0}' to resource group '{1}'.
```

Comment:
Hubs are only deployed to a RG today. The other scopes are used for other deployments. If this is the Toolkit module, we need to change "hub" here. If it's a Hubs module, then we can get rid of the others.

Reply:
fixed